<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeMain</class>
 <widget class="QMainWindow" name="EmployeMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>610</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_prenom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>660</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_salaire">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>710</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_poste">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>760</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_date">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>810</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ajouter">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>240</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>AJOUTER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_supprimer">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>630</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>SUPPRIMER</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>991</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_update">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>500</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>MODIFIER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>670</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Prenom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>620</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>NOM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>720</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Salaire</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>770</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Poste</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>820</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Date Embauche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>370</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>AFFICHER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sort">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>790</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: gray;
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Tri</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_export_pdf">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>840</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: gray;
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>1710</x>
      <y>940</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>940</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Recherche</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Image">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-180</y>
      <width>2061</width>
      <height>1321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>F:/Qt projet/Employee/5.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RDV">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
    background-color: white;
    color: rgb(0, 0, 0);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 0, 0);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>RDV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Vehicules">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
    background-color: white;
    color: rgb(0, 0, 0);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 0, 0);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Vehicules</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Services">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
    background-color: white;
    color: rgb(0, 0, 0);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 0, 0);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Services</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mdp">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>859</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>870</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>MDP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_toggleTheme">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>940</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Theme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_showLogs">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>940</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Historique</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>709</y>
      <width>991</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutChartView"/>
   </widget>
   <widget class="QPushButton" name="pushButton_RDV_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
    background-color: white;
    color: rgb(0, 0, 0);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 0, 0);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>EQUIPEMENT</string>
    </property>
   </widget>
   <zorder>label_Image</zorder>
   <zorder>lineEdit_ID</zorder>
   <zorder>lineEdit_nom</zorder>
   <zorder>lineEdit_prenom</zorder>
   <zorder>lineEdit_salaire</zorder>
   <zorder>lineEdit_poste</zorder>
   <zorder>lineEdit_date</zorder>
   <zorder>pushButton_ajouter</zorder>
   <zorder>pushButton_supprimer</zorder>
   <zorder>tableView</zorder>
   <zorder>pushButton_update</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>pushButton_refresh</zorder>
   <zorder>pushButton_sort</zorder>
   <zorder>pushButton_export_pdf</zorder>
   <zorder>lineEdit_search</zorder>
   <zorder>label_8</zorder>
   <zorder>pushButton_RDV</zorder>
   <zorder>pushButton_Vehicules</zorder>
   <zorder>pushButton_Services</zorder>
   <zorder>lineEdit_mdp</zorder>
   <zorder>label_9</zorder>
   <zorder>pushButton_toggleTheme</zorder>
   <zorder>pushButton_showLogs</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>pushButton_RDV_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
